flink-yarn/pom.xml

```diff
@@ -63,6 +63,11 @@ under the License.
 			<scope>test</scope>
 		</dependency>
 
+		<dependency>
+			<groupId>org.apache.hbase</groupId>
+			<artifactId>hbase-client</artifactId>
+			<version>${hbase.version}</version>
+		</dependency>
```

Comment:
This change is adding hbase as a dependency into Flink's fat jar (binary distribution).
The `flink-hbase` module is currently an optional module.